src/core/inductive.ml

```diff
@@ -55,17 +56,74 @@ let gen_ind_typ_codom : popt -> sym -> (tbox list -> tbox) -> string -> tbox =
     | _ -> fatal pos "The type of %a is not supported" pp_symbol ind_sym
   in aux [] !(ind_sym.sym_type)
 
+module T =
+  struct
+
+    (** [fold_app a [b1;...;bn]] returns (... ((a b1) b2) ...) bn. *)
+    let fold_app  : tbox -> tbox list -> tbox = List.fold_left _Appl
```

Comment:
Following the naming convention of terms.ml, this should be called _Appls or _Appl_list. Note that List.fold_left _Appl is probably used in other places. So the function _Appl_list could be defined in terms.ml.

Reply:
Ok done (in sr.ml too)